lib/spanner/command/reqrep.ex

```diff
@@ -57,12 +48,10 @@ defmodule Spanner.Command.Request do
 
   defp read_config(request, config_path) do
     [bundle, _cmd] = String.split(request.command, ":")
-    cmd_config_file = Path.join([config_path, bundle, "config.json"])
+    cmd_config_file = Path.join([config_path, bundle, Config.dynamic_file_name()])
```

Comment:
We don't just reference the attributes in this file. We also reference them in relay. We had "config.json" in several places throughout the code. It was a bit of a pain to track them all down. I moved it to one spot to make that easier if we ever change it again.